0001-kaa-protocol/README.md

```diff
@@ -7,83 +7,99 @@ contributors: Alexey Gamov <agamov@cybervisiontech.com>
 ---
 
 ## Introduction
-
-The document describes general guidelines, principles, and requirements for **Kaa protocol**.
+The document describes the general requirements, principles, design and guidelines for the **Kaa Protocol** (KP) version 1.
+KP is a standard protocol designed for connecting clients and endpoints (colloquially - devices) to a Kaa server.
 
 ## Requirements and constraints
-
+- KP must support asynchronous messaging.
 - Clients must be able to connect to Kaa server without using an SDK and with minimum custom implementation.
-- Encrypted and unencrypted channels should be supported.
-- Protocol should work over MQTT and CoAP.
-- Protocol must be extensible to support future extensions.
+- Encrypted and unencrypted channels must be supported.
+- KP must be able to work over MQTT v3.1 and later.
```

Comment:
That's just a note for you. The protocol itself is flexible enough to work with 3.1.